src/Decoder.cpp

```diff
@@ -106,6 +110,30 @@ namespace nbs {
         }
     }
 
+    Napi::Value Decoder::GetTypeIndex(const Napi::CallbackInfo& info) {
+        Napi::Env env = info.Env();
+
+        TypeSubtype type;
```

Comment:
Matching other places in this file:

```suggestion
        TypeSubtype typeSubtype;
```